### PR TITLE
DAWN-386 fix some of the test cases in special_accounts_tests

### PR DESCRIPTION
**Pull request by taokayan**
DAWN-386: fix some of the test cases in special_accounts_tests with master v3.
This is not done yet because the vote and register producer doesn't seems to work.

### CODE REVIEW

**Comment by taokayan** [comment]
Hi I've made another pull request. Hopefully this time it can be merged.

File: tests/tests/special_accounts_tests.cpp [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+      //    std::inserter(diff, diff.begin())
+      // );
+
+      // BOOST_CHECK_EQUAL(diff.size(), 0);
+
+      // const auto& producers_owner_authority = chain_db.get<permission_object, by_owner>(boost::make_tuple(config::producers_account_name, config::owner_name));
+      // BOOST_CHECK_EQUAL(producers_owner_authority.auth.threshold, 0);
+      // BOOST_CHECK_EQUAL(producers_owner_authority.auth.accounts.size(), 0);
+      // BOOST_CHECK_EQUAL(producers_owner_authority.auth.keys.size(), 0);
+
```
Comment (heifner) [inline tests/tests/special_accounts_tests.cpp]: Can you please remove all the commented out code.

File: tests/tests/special_accounts_tests.cpp [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+{ try {
+
+      tester test;
+      chain::chain_controller *control = test.control.get();
+      chain::database &chain1_db = control->get_mutable_database();
+      //chain::fork_database &fdb = control->_fork_db;
+      //chain::block_log &blog = control->_block_log;
+
+      //Make_Blockchain(chain1);
+
```
Comment (heifner) [inline tests/tests/special_accounts_tests.cpp]: Remove commented out code.

**Comment by brianjohnson5972** [comment]
@taokayan This is waiting on review changes that @heifner asked for.

**Comment by taokayan** [comment]
New pull request created for this.

**Pull request merged**